openlibrary/macros/LoanStatus.html

```diff
@@ -80,9 +80,60 @@
     <a href="$viewbook.replace('XXX', page.ocaid)" title="Use BookReader to read online" class="cta-btn cta-btn--available">Read</a>
 
 $elif (not page.get('ocaid') or page.is_access_restricted()) and editions_page:
-    <form>
-      <input type="submit" class="cta-btn" disabled value="No ebook available">
-    </form>
+    <!-- If user belongs to sponser-book group then show the sponser available button  -->
+    <!-- To implement: Using the policy api add more constraint that the book which is sponsorable
+    should only show this type of button  -->
+    $if (user and user.is_sponsor()):
+      <button id="myBtn" class="cta-btn" style="background-color: black">Sponser this ebook</button>
```

Comment:
Sponser -> Sponsor
Let's also capitalize the B (to give the text left-right balance) 
`Sponsor this eBook`

Reply:
Yup silly mistake